packages/remark-parse/lib/util/remove-indentation.js

```diff
@@ -56,17 +55,7 @@ function indentation(value, maximum) {
         index--;
       }
 
-      if (
-        trim(values[position]).length !== 0 &&
-        minIndent &&
-        index !== minIndent
-      ) {
-        padding = C_TAB;
-      } else {
-        padding = '';
-      }
```

Comment:
This logic for adding padding in a function that is supposed to be removing it is definitely incorrect. I've checked the failing test cases before and after this change, and this code block does more harm than good. There is only one test case whose output changes after making this change, and the previous fixture expected it to introduce a phantom `\t` into the output text paragraph. By removing this block, the phantom `\t` is removed and the test output is correct afaict.